action.yml

```diff
@@ -199,6 +199,26 @@ runs:
           cd /tmp/skaffold
           skaffold config set collect-metrics false
           skaffold deploy
+
+          if [ $? -eq 0 ]; then
+            # Publish Artifact Info on success
```

Comment:
Lets log an informational log message here prior to saving artifact info.

`echo "Saving artifact information for chart  $CHART_NAME"`